app/assets/javascripts/components/autocomplete.js

```diff
@@ -23,7 +23,17 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     new window.accessibleAutocomplete.enhanceSelectElement({ // eslint-disable-line no-new, new-cap
       selectElement: $select,
       minLength: 3,
-      showNoOptionsFound: true
+      showNoOptionsFound: true,
+      onConfirm: function (value) {
+        var category = $select.getAttribute('data-track-category')
+        var label = $select.getAttribute('data-track-label')
+        if (!category || !label) {
+          return
+        }
+
+        var action = value
+        window.GOVUK.analytics.trackEvent(category, action, { label })
```

Comment:
Could we potentially add another attribute called `data-track-select="true"`

```suggestion
        var trackSelect = $select.getAttribute('data-track-select') === "true"
        
        if (trackSelect) {
          var category = $select.getAttribute('data-track-category')
          var label = $select.getAttribute('data-track-label')

          window.GOVUK.analytics.trackEvent(category, value, { label: label })
        }
```

Reply:
What do you think about this vs. an explicit parameters on the autocomplete component for the tracking category and label?